src/panels/lovelace/entity-rows/types.ts

```diff
@@ -59,3 +60,9 @@ export interface ConditionalRowConfig extends EntityConfig {
   row: EntityConfig;
   conditions: Condition[];
 }
+export interface AttributeRowConfig extends EntityConfig {
+  attribute: string;
+  prefix?: string;
```

Comment:
Should we align this to be more like the Gauge Card Configuration? `Unit` and `name`? Maybe `attribute_name`

Also, is the default necessary for some attributes?

Reply:
I prefer what is currently set